portfolio/src/main/webapp/script.js

```diff
@@ -83,17 +84,20 @@ function getComments() {
 }
 
 function createCommentElement(commentEntity) {
-  const name = createH4Element(commentEntity.name);
-  const content = createSpanElement(commentEntity.content);
-  console.log(name);
-  console.log(content);
-
   const commentElement = document.createElement('li');
   commentElement.innerHTML = '';
 
+  const name = createH4Element(commentEntity.name);
   commentElement.appendChild(name);
+
+  const content = createSpanElement(commentEntity.content);
   commentElement.appendChild(content);
-  
+
+  if (typeof commentEntity.imageURL !== 'undefined') {
```

Comment:
nit: I'd simplify this to

`if (commentEntity.imageURL) {`

which not only covers `undefined`, but also `null` and `""` (JavaScript's type system is a bit archaic)

Reply:
Got it!